Cookbook.Infrastructur/Persistance/SqliteCommandTexts.cs

```diff
@@ -0,0 +1,121 @@
+﻿namespace Cookbook.Infrastructur.Persistance
+{
+    public static class SqliteCommandTexts //parameterize (create tables class, columns class)
```

Comment:
a comment az meg mindig el? ez todo vagy mit jelent ez?

Reply:
ez TODO akart lenni, de akartalak kerdezni h van e ertelme. Arra gondoltam h csinalok 
egy 

public class Tables
{
public const string Recipe = "Recipe";
}
meg hozza
public class RecipeTable
{
public const string  Title = "Title";
public const string Author = "Author"
}

es akkor ha valamikor valtozik a string egy helyen van, meg ha irom a sql querriket akkor felajanlja a VS es nem irom el veletlen

v nincs ertelme/nem szoktak? a column nev valtoztatast tudom h ritka de COnal Axel ajanlotta h csinaljak ilyet

Reply:
Nem, ez jo practice. Find reference is jobban mukodik mint a Ctrl + F. 
Azt nem talaltam meg pl hogy a tablakat hol hozod letre, akkor lenne igazan ertelme ha a create table-ben is tudod ujra hasznalni.